StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -0,0 +1,1223 @@
+
+**What was the best fit you managed to get?**
+
+<details><summary> What was the answer?</summary>
+
+A slope of approximately 3 should give the best answer (lowest sum of squares), which is
+3423.242.
+
+</details>
+
+
+**How confident are you that you found the best line?**
+
+It is hard to know by trial and error if you have found the 'best' line for the data.
+It is much easier, repeatable, and reliable to use a simple linear regression instead. The idea is the same as the app, but instead of trying until it looks good, the equation for simple linear regression is used and values for the unknown parameters are found using **maximum likelihood estimation**.
+
+</br>
+
+
+#### Interpreting the parameters
+
+Now we know what each of the three parameters in a simple linear regression
+mean, we can now think about interpreting them. 
+
+**Which of the three parameters do you think is most important for answering
+the research question "Does X influence Y?"?**
+
+</br>
+
+<details><summary>I had a go, now show me the answer.</summary></span>
+
+**The slope ($\beta$)** tells us the strength and direction the relationship between $X$
+and $Y$ is. While the linear regression model also estimates the intercept and the residual variance, these do not directly answer our question of interest. However, to make predictions, we will need all three parameters.
+
+</details>
+
+</br>
+
+### Worked example 
+
+</br>
+
+In the previous section of this worked example, we fit a simple linear 
+regression using the `lm()` function and looked at the estimates of 
+some parameters using the `coef()` function. In this section, we will use model theory to interpret what those parameters mean.
+
+</br>
+
+#### The intercept and slope
+
+We already know that the parameters of the intercept and slope control the position
+and steepness of the regression line. It is the estimates of these two parameters that we get from the `coef()` function. 
+
+</br>
+
+For our  model the estimates are:
+
+```{r, include = T, echo =T}
+coef(DiveModel)
+```
+
+</br>
+
+The intercept is 756 m and the slope of the relationship between body size and dive depth is -0.005. 
+
+</br>
+
+In this case, the intercept is not that interesting. It tells us the expected value of $Y$ (maximum dive depth) when $X$ (body size) = 0. It does not make a lot of biological sense to know the expected dive depth of a marine mammal that weighs 0 kg. But, sometimes it can make sense to know the value $Y$ when $X$ is 0, for example if $X$ was temperature.
+
+</br>
+
+The slope on the other hand is interesting. It tells us the direction and strength of the relationship between body size and maximum dive depth. In this case our model estimates a negative slope and for every increase of 1 kg in body size, marine mammals will have a maximum dive depth that is 0.005 m less deep. In other words, there is a negative relationship between body weight and dive depth and as $X$ increases $Y$ decreases (Figure 4).
+
```

Comment:
I think the second sentence is a bit complicated. This is just a suggestion.

In this case our model estimates a negative slope.
This means if the body size of marine mammals increase by 1kg the maximum dive depth decreases by 0.005 m.